subjects/java/checkpoints/good-bye-mars/README.md

```diff
@@ -0,0 +1,36 @@
+## GoodbyeMars
+
+### Instructions
+
+In a file named `GoodbyeMars.java` write a function `goodbyeMars` that returns the string 'Good bye Mars !'.
+
+### Expected Functions
+
+```java
+public class GoodbyeMars {
+    public static String goodbyeMars() {
+        // your code here
+    }
+}
+```
+
+### Usage
+
+Here is a possible `ExerciseRunner.java` to test your function :
+
+```java
+public class ExerciseRunner {
+    public static void main(String[] args) {
+        System.out.println(GoodbyeMars.goodbyeMars());
+    }
+}
+```
+
+and its output :
+
+```shell
+$ javac *.java -d build
+$ java -cp build ExerciseRunner
+Good bye Mars !
```

Comment:
`Goodbye` should be all together.
Is there any reason why the space between `Mars !`?

Reply:
Is there any reason why the space between Mars !? : no reason 
Goodbye should be all together.: yeah i change it